utils/index.js

```diff
@@ -32,13 +32,13 @@ export const mapRepoFromStorageToUi = (persistedData) => {
 };
 
 /**
- * Reads the persisted repo data from a JSON file
+ * Reads the persisted data from a JSON file
```

Comment:
I guess `Reads data from a JSON file` might be more accurate, but it's not a big deal

Reply:
I agree - I've updated the description.